zuul.ansible.d/nodesets.yaml

```diff
@@ -575,6 +575,21 @@
         nodes:
           - fedora-35
 
+- nodeset:
+    name: vsrx3-18.4R1-python39
```

Comment:
We have decided to maintain a generic name for the nodes and not to use the python versions along with it. Can you please remove pyhton39 from the name and modify the references?